packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -318,24 +337,29 @@ const NuovaDelega = () => {
                             <Grid item xs={isMobile ? 12 : 6} className={classes.margin}>
                               {values.selectTuttiEntiOrSelezionati === 'entiSelezionati' && (
                                 <FormControl fullWidth>
-                                  <CustomDropdown
+                                  <Autocomplete
                                     id="ente-select"
-                                    label={t('nuovaDelega.form.selectEntities')}
+                                    multiple
+                                    options={entities}
                                     fullWidth
-                                    value={values.enteSelect.uniqueIdentifier}
-                                    onChange={(event: SelectChangeEvent<string>) => {
-                                      setFieldValue('enteSelect', {
-                                        name: event.target.name,
-                                        uniqueIdentifier: event.target.value,
-                                      });
+                                    autoComplete
+                                    getOptionLabel={getOptionLabel}
+                                    isOptionEqualToValue={(option, value) =>
+                                      option.name === value.name
+                                    }
+                                    onChange={(_event: any, newValue: Array<Party>) => {
+                                      setFieldValue('enteSelect', newValue);
                                     }}
-                                  >
-                                    {entities.map((entity) => (
-                                      <MenuItem value={entity.id} key={entity.id}>
-                                        <DropDownPartyMenuItem name={entity.name} />
-                                      </MenuItem>
-                                    ))}
-                                  </CustomDropdown>
+                                    inputValue={inputValue}
+                                    onInputChange={(_event, newInputValue) =>
+                                      handleChangeInput(newInputValue)
+                                    }
+                                    filterOptions={handleFilterAutocomplete}
+                                    renderOption={renderOption}
+                                    renderInput={(params) => (
+                                      <TextField {...params} label="Seleziona enti" />
```

Comment:
use the translation function for the label attribute:
{ t('nuovaDelega.form.selectEntities') }

Reply:
do it